orca/orca-bakery/src/main/java/com/netflix/spinnaker/orca/bakery/tasks/manifests/CreateBakeManifestTask.java

```diff
@@ -84,7 +85,7 @@ public TaskResult execute(@Nonnull StageExecution stage) {
       throw new IllegalStateException(
           "A BakeryService must be configured in order to run a Bake Manifest task.");
     }
-
+    ObjectMapper mapper = new ObjectMapper();
```

Comment:
this is a "please don't do this" - object mappers shouldn't be created on execution request, and misses a number of properties.  Should be an autowired component & globally configured as much as possible.